_migration-assistant/migration-phases/planning-your-migration/adjusting-settings-during-migration.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Adjusting settings during migration
+nav_order: 60
+parent: Planning your migration
+grand_parent: Migration phases
+redirect_from:
+  - /migration-assistant/migration-phases/adjusting-settings-during-migration/
+---
+
+# Adjusting settings during migration
+
+During a migration there is an opportunity change names and paths of index and data structures that are not modifiable after they are declared. This page provides a cookbook of different scenarios and templates that can be used to make these adjustments.
+
+### Rename an index
+
+> [!NOTE]  
+> We recommend creating an alias to an index instead of renaming them, if an alias is created on the source cluster the migration process will copy the alias over as well. See create an alias [documentation](https://opensearch.org/docs/latest/api-reference/index-apis/update-alias/).
+
+#### Migration with a snapshot
+
+1. Navigate to the bootstrap box and open the `cdk.context.json` with vim
+2. Add/Update the key `reindexFromSnapshotExtraArgs` to include `--doc-transformer-config-file /shared-logs-output/rfs-transform.json`
+3. Redeploy the Migration Assistant 
+4. Navigate on to the Migration Assistant console
+5. Create a file with `vim /shared-logs-output/rfs-transform.json`
+6. Add the following to the file:
+```json
+[
+  {
+    "JsonConditionalTransformerProvider": [
```

Comment:
@AndreKurait - do you think we'll keep conditional transformers in place once we support writing transformations in scripting languages (e.g. javascript)?

Reply:
This code is functional - when JavaScript becomes available we can always edit this example to be cleaner.

Reply:
We've got approval on the UPL license yesterday, so I'd expect that Javascript will be available imminently (and jolt transforms will be removed probably at the 2.2 release).  Please replace these instructions with javascript ones so that this description isn't DOA and confusing to customers.